html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -12,7 +12,9 @@ use pf::authentication;
 use HTML::Entities;
 use List::MoreUtils qw(any);
 use pf::config;
+use pf::person qw(person_modify @FIELDS);
```

Comment:
There's no need to import '@FIELDS' since you use the complete name in line 17 and it is the right way to do :)